pom.xml

```diff
@@ -216,6 +216,33 @@ The MIT License (MIT)
               </execution>
             </executions>
           </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <version>3.0.0</version>
+            <configuration>
+              <tags>
+                <tag>
+                  <name>todo</name>
+                  <placement>a</placement>
```

Comment:
@Isammoc please remove `<head>` and set `<placement>` to "X" to both the `todo` config and the `checkstyle` config. I don't want them showing up in javadocs.

Reply:
@llorllale Done. 
IMHO, that would be better to see that a method is still a work in progress (with the todo showing in javadocs) And a version should not be tagged if a puzzle exist in the codebase.

But you are the `ARC` here ^^.

Reply:
@Isammoc that's ok, any issues raised would be duplicates of existing `@todo`s.